### PR TITLE
Migration list queries resource_queue subset of tests to ICG

### PR DESCRIPTION
**Pull request by oarap**
The PR adds qp_resource_queue subset of tests to ICG. There are some tests that needs further special processing. 
query 6 is a duplicate, 16,18,19,20 are skipped, 11,12,13,14,15 and 21 needs special processing.

Ed & Omer

### CODE REVIEW

File: src/test/regress/input/qp_resource_queue.source [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+create schema qp_resource_queue;
+set search_path to qp_resource_queue;
+
+drop table if exists tenk1;
+
+create table tenk1 (unique1 int4,unique2 int4, two int4,four int4,ten int4,twenty int4,
+                                hundred int4,thousand int4,twothousand int4,fivethous int4,tenthous int4,
+                                odd int4,even int4,stringu1 name,stringu2 name,string4 name);
+
+copy tenk1 from '@abs_srcdir@/data/tenk.data';
```
Comment (vraghavan78) [inline src/test/regress/input/qp_resource_queue.source]: did you folks add the tenk.data file?

Comment (oarap) [inline src/test/regress/input/qp_resource_queue.source]: It was already there in the data folder and I diffed it with the one in the cdbfast and they were identical.

Comment (vraghavan78) [inline src/test/regress/input/qp_resource_queue.source]: are the test the same? do we even need it?

Comment (oarap) [inline src/test/regress/input/qp_resource_queue.source]: No tests are totally different. Actually there are there different test suites in addition to this one utilizing this data file.

**Comment by oarap** [comment]
@xinzweb @hsyuan Please take a look into this PR whenever you have time.

**Comment by hlinnaka** [comment]
These tests seem to be duplicates of the existing "resource_queue" test.

**Comment by oarap** [comment]
Yes, this seems to be a duplicate test of the resource queue tests that are already in ICG, closing the pull request.

**Pull request closed without merging**